.travis.yml

```diff
@@ -87,6 +87,15 @@ jobs:
           - STARR=$(sed -n 's/^maven\.version\.number=//p' buildcharacter.properties) && echo $STARR
           - sbt -Dscala.build.compileWithDotty=true library/compile
 
+      - state: test
```

Comment:
stage.  Or omit it as it'll copy the previous job's stage.

Reply:
arg